doc/sphinx/source/gensidebar.py

```diff
@@ -47,22 +47,38 @@ def _write(project, desc, link, mapping=conf['intersphinx_mapping']):
             args = desc, link
             lines.append("    %s <%s>" % args)
 
+    def _header(project, text):
+        if project == conf_api or do_gen:
+            lines.extend([".. toctree::", "   :maxdepth: 2"])
+            lines.extend(["   :caption: %s" % text, ""])
+
     #
     # Specify the sidebar contents here
     #
 
-    _toctree()
-    _write("esmvaltool", "Preface", "preface/index")
+    _header("esmvaltool", "ESMValTool")
+    _write("esmvaltool", "Introduction", "introduction")
     _write("esmvaltool", "Gallery", "gallery")
```

Comment:
I would put the gallery after Getting started.

Reply:
In the proposal in the issue it is before Getting started, that's why I put it where it is. I would be happy to change it if, but not if other people think it should be in this location. Maybe we can discuss this in the issue?

Reply:
Discussion now in https://github.com/ESMValGroup/ESMValTool/issues/1574

Reply:
Done now